src/cjs-download/index.ts

```diff
@@ -4,13 +4,27 @@ import consistentSort from "../lib/consistentSort"
 import convertOds from "./convertOds"
 import downloadCjsFile from "./downloadCjsFile"
 import getDownloadUrl from "./getDownloadUrl"
+import mapOffenceCodeData from "../map-data/mapOffenceCodeData"
+
+type OffenceCodeRecords = {
+  cjsCode: string
+  description?: string
+  homeOfficeClassification?: string | null
+  notifiableToHo?: string
+  recordCreated?: number[]
+  source?: string
+  offenceCategory?: string
+  offenceTitle?: string
+  recordableOnPnc?: string
```

Comment:
shouldn't this be a boolean? I thought we were changing all of the underlying data